### PR TITLE
Rename the datastore-utils module to core-utils.

### PR DESCRIPTION
**Pull request by SPalominos**
Rename the datastore-utils module to core-utils.

### CODE REVIEW

**Comment by ebocher** [comment]
As Core-utils will contain all core functions and not only datastore one, it must be as a separate and top module

**Comment by SPalominos** [comment]
> As Core-utils will contain all core functions and not only datastore one, it must be as a separate and top module

Done

File: geometry-utils/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <!-- The Basics -->
+    <!-- The Basics -->0297639747
```
Comment (ebocher) [inline geometry-utils/pom.xml]: ?

File: osm-utils/src/test/groovy/org/orbisgis/osm_utils/utils/DataUtilsTest.groovy [hunk 2]
```diff
@@ -37,11 +37,14 @@
...
+import org.h2gis.utilities.JDBCUtilities
 import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
 
 import java.sql.Connection
 
+import static org.h2gis.utilities.JDBCUtilities.*
+
 class DataUtilsTest {
     private static Connection CONNECTION
```
Comment (ebocher) [inline osm-utils/src/test/groovy/org/orbisgis/osm_utils/utils/DataUtilsTest.groovy:49]: maj ?

**Pull request merged**